commitApiDoc.js

```diff
@@ -0,0 +1,34 @@
+const commitApiDoc = async (options) => {
+  const {
+    octokit,
+    owner,
+    repo,
+    ref,
+    file
+  } = options;
+  const { name, content } = file;
+
+  const defaultParams = {
+    owner,
+    repo,
+    path: name,
+    message: `Commiting file ${name}`,
+    content,
+    branch: ref,
+  };
+
+  const params = await octokit.repos.getContent({
+    owner,
+    repo,
+    ref,
+    path: name,
+  }).then(({ data }) => Promise.resolve({
+    ...defaultParams,
+    sha: data.sha,
+  }))
+  .catch(() => Promise.resolve(defaultParams));
```

Comment:
Nos callback do `.then` e do `.catch` não precisa retornar uma `Promise`. Só retornando o valor já é suficiente.

https://github.com/microsoft/TypeScript/blob/68b0323b72bf51236d8cd181ed94ba9a63f933fd/lib/lib.es5.d.ts#L1446-L1461

```suggestion
  }).then(({ data }) => ({
    ...defaultParams,
    sha: data.sha,
  }))
  .catch(() => defaultParams);
```